setup.py

```diff
@@ -16,7 +16,7 @@
     long_description_content_type="text/markdown",
     url="https://github.com/ladybug-tools/honeybee-energy",
     packages=setuptools.find_packages(exclude=["tests"]),
-    install_requires=['git+https://github.com/ladybug-tools/honeybee-core.git'],
+    install_requires=["honeybee-core"],
```

Comment:
I didn't realize this could work.  Can I similarly swap out the ladybug core reference to the github URL of ladybug_geometry for `ladybug-geometry`?

Reply:
Yes and I think that is the right thing to do.

Reply:
I should also add that this will not work currently as deployment of honeybee-core failed but it will work once honeybee-core is available on PyPI.